/usr/include/linux/usb.h doesn't exist anymore, ezusb can use
definitions from linux/usb/ch9.h, see
http://qa.mandriva.com/show_bug.cgi?id=36214

diff -p -up fxload-2002_04_11/ezusb.c.orig fxload-2002_04_11/ezusb.c
--- fxload-2002_04_11/ezusb.c.orig	2008-01-09 10:18:00.000000000 -0500
+++ fxload-2002_04_11/ezusb.c	2008-01-09 10:19:04.000000000 -0500
@@ -30,7 +30,7 @@
 # include  <asm/types.h>
 
 # include  <linux/version.h>
-# include  <linux/usb.h>
+# include  <linux/usb/ch9.h>
 # include  <linux/usbdevice_fs.h>
 
 # include "ezusb.h"
